backend/app/llm_utils/completion.py

```diff
@@ -56,7 +99,7 @@ async def _ask_llm_async(user_message: str, system_message: str, model: str) ->
         "messages": messages,
         "temperature": 0,
         "max_tokens": 1024,
-        "response_format": {"type": "json_object"},
+        # "response_format": {"type": "json_object"},
```

Comment:
@sidravi1 what is the status of this code chunk? I see we've commented it out for now. Should it be reintroduced when working w/ 4o only?

Reply:
ah yes. It doesn't work across all models so I have commented it out. 4o works pretty well without it - still give valid json. I think we can uncomment it when we do validation if performance is not great.